parsl/executors/high_throughput/executor.py

```diff
@@ -501,7 +501,9 @@ def submit(self, func, *args, **kwargs):
         self._task_counter += 1
         task_id = self._task_counter
 
-        logger.debug("Pushing function {} to queue with args {}".format(func, args))
+        # handle people sending blobs gracefully
+        args_to_print = tuple([arg if len(repr(arg)) < 100 else (repr(arg)[:100] + '...') for arg in args])
```

Comment:
What do you think about creating args_to_print conditional on debug logs being enabled?

Reply:
Makes perfect sense. Done.